core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -121,21 +125,26 @@ class AdminClient(val time: Time,
     listAllGroupsFlattened.filter(_.protocolType == ConsumerProtocol.PROTOCOL_TYPE)
   }
 
-  def describeGroup(groupId: String): GroupSummary = {
-    val coordinator = findCoordinator(groupId)
-    val responseBody = send(coordinator, ApiKeys.DESCRIBE_GROUPS, new DescribeGroupsRequest(List(groupId).asJava))
-    val response = new DescribeGroupsResponse(responseBody)
-    val metadata = response.groups().get(groupId)
-    if (metadata == null)
-      throw new KafkaException(s"Response from broker contained no metadata for group $groupId")
-
-    Errors.forCode(metadata.errorCode()).maybeThrow()
-    val members = metadata.members().map { member =>
-      val metadata = Utils.readBytes(member.memberMetadata())
-      val assignment = Utils.readBytes(member.memberAssignment())
-      MemberSummary(member.memberId(), member.clientId(), member.clientHost(), metadata, assignment)
-    }.toList
-    GroupSummary(metadata.state(), metadata.protocolType(), metadata.protocol(), members)
+  def describeGroup(groupId: String): Option[GroupSummary] = {
+    findCoordinator(groupId) match {
+      case None => 
+        println("Could not find coordinator for group %s, which implies that one of the consumer offsets partitions may be offline or in the process of being created if this is a new cluster.".format(groupId))
```

Comment:
This is odd given that we are using exceptions everywhere else. If we want to avoid the stacktrace, we should handle it wherever we catch the exceptions.